aiohue/sensors.py

```diff
@@ -0,0 +1,55 @@
+from .api import APIItems
+
+
+class Sensors(APIItems):
+    """Represents Hue Sensors.
+
+    https://developers.meethue.com/documentation/sensors-api
+    """
+
+    def __init__(self, raw, request):
+        super().__init__(raw, request, 'sensors', Sensor)
+
+
+class Sensor:
+    """Represents a Hue sensor."""
+    def __init__(self, id, raw, request):
+        self.id = id
+        self.raw = raw
+        self._request = request
+
+    @property
+    def name(self):
+        return self.raw['name']
+
+    @property
+    def type(self):
+        return self.raw['type']
+
+    @property
+    def modelid(self):
+        return self.raw['modelid']
+
+    @property
+    def manufacturername(self):
+        return self.raw['manufacturername']
+
+    @property
+    def productname(self):
+        return self.raw.get('productname')
+
+    @property
+    def uniqueid(self):
+        return self.raw.get('uniqueid')
+
+    @property
+    def swversion(self):
+        return self.raw.get('swversion')
```

Comment:
This one is always available as per https://developers.meethue.com/documentation/supported-sensors#generalSensorResource

Reply:
This one says it's "Not applicable for ZGP devices" - so the Hue Tap.